crypto/cipher/aes_icm.c

```diff
@@ -199,8 +199,8 @@ static srtp_err_status_t srtp_aes_icm_context_init (void *cv, const uint8_t *key
 
     copy_len = c->key_size - base_key_len;
     /* force last two octets of the offset to be left zero (for srtp compatibility) */
-    if (copy_len > 14) {
-        copy_len = 14;
+    if (copy_len > SRTP_SALT_LEN) {
```

Comment:
Can't say the check for >14 is wrong, since I assume there might be reasons.  When using GCM, for example, the salt is 12 octets, but when performing RTP header encryption, ICM is used.  So, the user will provide a 128-bit key and a 14-octet salt.  The first 12 octets of the salt are used in GCM and the other two are used in header encryption.  At least I think that's how it was implemented.

But, I'm just unsure of what examples exist where the salt is < 14.  Maybe it's valid, but not sure.  Certainly, I do know ICM needs a 14 octet salt.